tests/base_test_mixins.py

```diff
@@ -513,3 +735,50 @@ def debug_print_response_messages(self, response):
             print("-----------message end-------------\n")
         except KeyError:
             print("\n-------no message----------")
+
+
+class InProcessFakeContainerTestMixin(object):
```

Comment:
Sorry--not in-process at all. It's a subprocess the way it's currently implemented. (Btw, how come youd didn't do in-process? I.e. just a function call?)

Reply:
Sorry, do you mean the function `main()` in `runpy`?  My reason (might be wrong), was  `runpy` is not in a package, and it's hard to import from (though not impossible). And running a subprocess is easy to terminate in TestCase. Or is there an easier way?

Reply:
More over, the subprocess it self can be tested (and included in coverage).

Reply:
That's fair. I don't mind the subprocess. But then this should be renamed to "`SubProcess...`".